uiowa/uiowa
# Admissions - 2 Plus 2 Major content type with Community Colleges term display

Resolves #5003 
Resolves https://github.com/uiowa/uiowa/issues/5132

As a 2plus2 transfer student, I would like to be able to see what specific checkpoints and courses I need to complete at my community college to meet the University of Iowa requirements for that major.

As a 2plus2 transfer student, I can see all 2plus2 majors within a participating community college.

# How to test

- `ddev blt ds --site=admissions.uiowa.edu`
- `ddev drush @admissions.local uli`
- As a webmaster a term for the community college taxonomy so you'll have a college to reference. Go on, fill it out with some content.
- As a webmaster add at least Major (2 Plus 2) piece of content. Fill out all of the main fields. Save.
- Check that the title of the content is '(chosen area of study) Major Through (chosen community college)'.
- Also check that the community college checkpoints field displays as '(chosen community college) checkpoints'.
- Notice the breadcrumb links to the community college you selected. Navigate there and see the related AoS titles show within the taxonomy term fielded content with a contextually aware heading that includes the term name. If there is more than one 2plus2 major referencing the community college, it should show in alphabetical order.
- See if you can break anything.
